awscdk-minecraft/src/zip-directory.py

```diff
@@ -0,0 +1,40 @@
+import os
+from zipfile import ZipFile
+
+def collect_file_paths(target_dir):
+    # Initialize a list for adding filepath strings to
+    file_paths = []
+    # Walk through the target directory and subdirectories generating file paths
+    for root, directories, files in os.walk(target_dir):
+        # For each directory or sub directory create a full length path for each file
+        for filename in files:
+            # Create a complete file path
+            filepath = os.path.join(root, filename)
+            # Append the file path to the list of filepaths
+            file_paths.append(filepath)
+
+    # Output the list of complete file paths
+    return file_paths
+
+def main():
+    # Path to target folder
+    target_dir = './'
+
+    # Grab all file paths to be zipped
+    file_paths = collect_file_paths(target_dir)
+
+    # Print that paths of the all the files to be zipped
+    print('These files will be zipped:\n')
+    for paths in file_paths:
```

Comment:
Very minor, but maybe slightly more descriptive if we call this variable `path` instead of `paths` since they are singular